frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.mdx

```diff
@@ -0,0 +1,104 @@
+import { Canvas, Meta } from '@storybook/blocks';
+import { Heading, Paragraph } from '@digdir/design-system-react';
+import * as StudioTableLocalPaginationStories from './StudioTableLocalPagination.stories';
+
+<Meta of={StudioTableLocalPaginationStories} />
+
+<Heading level={1} size='small'>
+  StudioTableLocalPagination
+</Heading>
+<Paragraph>StudioTableLocalPagination is a wrapper for StudioTableRemotePagination.</Paragraph>
```

Comment:
We should avoid writing about implementation details. This component can also be used without being a wrapper. This component should not have a relationship with which other components use it.


Potentially, there might be several components built upon this one, making it difficult to keep this documentation updated. Additionally, it doesn't provide much value for the user of this component.

I think it's more meaningful to write the documentation like this:

```suggestion
<Paragraph>The StudioTableLocalPagination component handles pagination internally, eliminating the need for manual control. It seamlessly manages pagination logic for you. </Paragraph>
```

What do you think about this suggestion? 😊

Reply:
That is better, fixed 😊